build.gradle

```diff
@@ -19,10 +15,12 @@ subprojects {
 
     repositories {
         google()
+        mavenCentral()
         jcenter()
```

Comment:
Should we just go ahead and remove this now

Reply:
We can't, unfortunately. AGP still depends on one transitive artifact that is only available in JCenter. Presumably when AGP 4.2 comes out, that dependency will be gone and we can remove the repo here.